src/consumer/base_consumer.rs

```diff
@@ -284,6 +284,18 @@ impl<C: ConsumerContext> Consumer<C> for BaseConsumer<C> {
         }
     }
 
+    fn committed_offsets<T: Into<Option<Duration>>>(&self, mut tpl: TopicPartitionList, timeout: T) -> KafkaResult<TopicPartitionList> {
+        let committed_error = unsafe {
```

Comment:
This code seems to be the same as in `committed` (apart from the `.ptr()` which shouldn't be a problem). Would it make sense to refactor `committed` to call `committed_offsets`?